config.rb

```diff
@@ -41,23 +46,9 @@ def ruby_shops
   activate :minify_javascript
   activate :minify_html
   activate :gzip
-  activate :imageoptim
   activate :asset_hash
-
-  require_relative "./lib/build_cleaner"
-  activate :build_cleaner
-end
-
-activate :google_analytics do |ga|
```

Comment:
We were using a plugin for that before. If you remove it, I guess it's because it's no more available for the v4?